src/Facade/Services/GoodsInLogFacadeService.cs

```diff
@@ -0,0 +1,95 @@
+namespace Linn.Stores2.Facade.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq.Expressions;
+    using System.Threading.Tasks;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Persistence;
+    using Linn.Stores2.Domain.LinnApps;
+    using Linn.Stores2.Domain.LinnApps.GoodsIn;
+    using Linn.Stores2.Facade.Common;
+    using Linn.Stores2.Resources;
+    using Linn.Stores2.Resources.GoodsIn;
+
+    public class GoodsInLogFacadeService : AsyncFacadeService<GoodsInLogEntry, int, GoodsInLogEntryResource, GoodsInLogEntryResource, GoodsInLogEntrySearchResource>
```

Comment:
i suppose in theory this should really adhere to an interface that doesn't support adding, updating and removing since goods in log is never directly manipulated from this app

Reply:
(you can just write your own IGoodsInFacadeService interface with one public GetReport() method if you go down the reporting route)